lib/plugins/slack/attachements.js

```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
       }
     }
 
+    let message = '';
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
```

Comment:
https://github.com/sitespeedio/sitespeed.io/issues/1527 let us discuss it there!